src/network/bmproto.py

```diff
@@ -70,7 +88,10 @@ def bm_proto_reset(self):
         self.object = None
 
     def state_bm_header(self):
-        self.magic, self.command, self.payloadLength, self.checksum = protocol.Header.unpack(self.read_buf[:protocol.Header.size])
+        """Predicate (with logging) to indicate the prescence of a header"""
```

Comment:
It doesn't indicate a header presence, it expects it, according to the protocol specification.